src/views/Joseki/Joseki.tsx

```diff
@@ -0,0 +1,1174 @@
+/*
+ * Copyright (C) 2012-2017  Online-Go.com
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* A page for looking up and playing against josekis */
+
+import * as React from "react";
+import * as ReactMarkdown from "react-markdown";
```

Comment:
Could you see if you can switch this to our Markdown component? We do some html filtering in there, and it would make our markdown systems homogenous which I always prefer. It should be pretty much a drop in replacement, see https://github.com/online-go/online-go.com/blob/devel/src/views/User/User.tsx#L44 and https://github.com/online-go/online-go.com/blob/devel/src/views/User/User.tsx#L755 for usage, it's pretty much identical.

Reply:
Sure, as you say: it was a direct swap :)  Done!